### PR TITLE
[Wikipedia] Added show details view for wikipedia page.

### PR DESCRIPTION
**Pull request by shubzkothekar**
## Description

- Added show details view and action for wikipedia page.
- Added wikipedia page thumbnail as icon for pages list while searching pages.

## Screencast

Screenshot 2022-08-19 at 5 41 27 PM

Screenshot 2022-08-19 at 5 41 44 PM

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
@vimtor do you mind checking this 🙂

File: extensions/wikipedia/src/search-page.tsx [hunk 1]
```diff
@@ -21,16 +22,22 @@ export default function SearchPage() {
...
       title={title}
-      subtitle={extract}
+      subtitle={data?.description}
       actions={
         <ActionPanel>
+          <PushAction
+            icon={Icon.Window}
+            title={"Show Details"}
+            target={<ShowDetailsPage title={title} extract={data?.extract} description={data?.description} />}
+          />
```
Comment (vimtor) [inline extensions/wikipedia/src/search-page.tsx:40]: I think this should be either the secondary action or we should add a preference to toggle it.

File: extensions/wikipedia/src/show-details-page.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  `;
+
+  return (
+    <Detail
+      navigationTitle={props.title}
+      markdown={markdown}
+      actions={
+        <ActionPanel>
+          <OpenInBrowserAction url={`https://wikipedia.org/wiki/${encodeTitle(props.title)}`} />
+          <CopyToClipboardAction title="Copy URL" content={`https://wikipedia.org/wiki/${encodeTitle(props.title)}`} />
```
Comment (vimtor) [inline extensions/wikipedia/src/show-details-page.tsx:24]: To match the `search-page` behaviour I would like to have `⌘+.` or `⌘+Shift+C` for copying the URL.

**Review by vimtor — CHANGES_REQUESTED** [review]

File: extensions/wikipedia/src/search-page.tsx (hunk 1 above)
Comment (shubzkothekar) [reply extensions/wikipedia/src/search-page.tsx:41]: Works for me, I have moved `Show Details` action to secondary action.

File: extensions/wikipedia/src/show-details-page.tsx (hunk 2 above)
Comment (shubzkothekar) [reply extensions/wikipedia/src/show-details-page.tsx]: I have added `⌘` + `.` shortcut in `Show Details` page

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**